tutorials/tutorial2.ipynb

```diff
@@ -197,178 +197,51 @@
-      "\u001b[33mFinished building.\u001b[0m\n",
-      "Inference -------------------- [ 0/29] \n",
-      "\u001b[33mInference time taken:  189.338s (Load    9.037s,Prediction  179.972s)\u001b[0m\n",
-      "INFO:2020-06-10 16:40:22,944:AtacWorks-main] Waiting for writer to finish...\n",
-      "Writing the output to bigwig files\n",
-      "Saving config file to ./infer_config.yaml...\u001b[0m\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "!atacworks denoise \\\n",
-    "    --input_files NK.50_cells.h5 \\\n",
+    "    --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \\\n",
+    "    --interval_size 50000 \\\n",
     "    --genome $atacworks/data/reference/hg19.auto.sizes \\\n",
-    "    --config configs/infer_config.yaml \\\n",
-    "    --config_mparams configs/model_structure.yaml"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Note: `infer_config.yaml` is set up to use multiple GPUs. If you are using a single GPU, edit `infer_config.yaml` to change the line `gpu: \"None\"` to read `gpu: 0`."
+    "    --weights_path ./models/model.pth.tar \\\n",
+    "    --out_home \"./\" \\\n",
+    "    --exp_name \"ataworks_denoise\" \\\n",
+    "    --gpu_idx 0"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The inference results will be saved in the folder `output_latest`. This folder will contain four files: \n",
-    "1. `NK_inferred.track.bedGraph` \n",
-    "2. `NK_inferred.track.bw` \n",
-    "3. `NK_inferred.peaks.bedGraph`. \n",
-    "4. `NK_inferred.peaks.bw`\n",
+    "The inference results will be saved in the folder `atacworks_denoise_latest`. This folder will contain four files: \n",
+    "1. `NK_infer.track.bedGraph` \n",
+    "2. `NK_infer.track.bw` \n",
+    "3. `NK_infer.peaks.bedGraph`. \n",
+    "4. `NK_infer.peaks.bw`\n",
     "\n",
-    "`NK_inferred.track.bedGraph` and `NK_inferred.track.bw` contain the denoised ATAC-seq track. `NK_inferred.peaks.bedGraph` and `NK_inferred.peaks.bw` contain the positions in the genome that are designated as peaks (the model predicts that the probability of these positions being part of a peak is at least 0.5)\n",
+    "`NK_infer.track.bedGraph` and `NK_infer.track.bw` contain the denoised ATAC-seq track. `NK_infer.peaks.bedGraph` and `NK_infer.peaks.bw` contain the positions in the genome that are designated as peaks (the model predicts that the probability of these positions being part of a peak is at least 0.5)\n",
     "\n",
-    "To change any of the parameters for inference with the deep learning model, you can edit the parameters in `configs/infer_config.yaml` or `configs/model_structure.yaml` and run the command above. \n",
+    "To change any of the parameters for inference with the deep learning model, you can edit the parameters in `$atacworks/configs/infer_config.yaml` or `$atacworks/configs/model_structure.yaml` and run the command above. \n",
```

Comment:
The user should be editing the config files they downloaded rather than the default config files

Reply:
done.